src/org/zaproxy/zap/extension/authstats/ExtensionAuthStats.java

```diff
@@ -157,4 +132,36 @@ public void onHttpResponseReceive(HttpMessage msg, int initiator, HttpSender sen
             log.error(e.getMessage(), e);
         }
     }
+
+    private static void updateAuthIndicatorStats(HttpMessage msg, String site, String prefix, Context context) {
+        AuthenticationMethod auth = context.getAuthenticationMethod();
+        if (auth == null) {
+            // No authentication method no indicators.
+            Stats.incCounter(site, prefix + "noindicator");
```

Comment:
Is it worth differentiating noauth vs noindicator? If not then combine the conditionals?

Reply:
Yeah, probably better to differentiate.